src/test/java/com/wikia/webdriver/testcases/mercurytests/RecentWikiActivityTest.java

```diff
@@ -18,6 +18,7 @@
 public class RecentWikiActivityTest extends NewTestTemplate {
 
   @Test(groups = "mercury_recentWikiActivity_anonNavigateToRWAfromMenu")
+  @Execute(mockAds = "true")
```

Comment:
I know that this is the interstitial, so as long as that ad behavior is verified differently, LGTM